### PR TITLE
Description update

### PR DESCRIPTION
**Pull request by Monstarules**
Should be good. Covers everything I said in #general-dev channel on the TPPcord.

### CODE REVIEW

File: description/active/universal/15 Credits.md [hunk 1]
```diff
@@ -31,15 +31,14 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
 - Fenris2142
 - ferraro
 - Kazowar
-- Ravendwyr
-- ScruffyKip
```
Comment (m4-used-rollout) [inline description/active/universal/15 Credits.md:35]: Why?

File: description/active/universal/15 Credits.md [hunk 2]
```diff
@@ -91,31 +91,32 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
 - Beesafree
 - BloodlessNS
 - ChauzuVGC
-- ScruffyKip
 - KoolboymanAdam
-- LightningXCE
```
Comment (m4-used-rollout) [inline description/active/universal/15 Credits.md:96]: Why?

File: description/active/universal/15 Credits.md [hunk 3]
```diff
@@ -91,31 +91,32 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
...
 - Tustin2121
 - Umericchi
 *****
-## Rom Testers
+## ROM Testers
 - Cosmog
 - Deadinsky
 - Faithfulforce
-- ScruffyKip
-- LightningXCE
```
Comment (m4-used-rollout) [inline description/active/universal/15 Credits.md:114]: Why?

File: description/active/universal/15 Credits.md [hunk 4]
```diff
@@ -152,15 +153,19 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
...
 - [Winamp **\(v5.666\)**](http://forums.winamp.com/showthread.php?t=373755) + [vgmstream plugin](https://github.com/losnoco/vgmstream)
 *****
 ## Commentary
-- MathyFurret
-- GeForceFly
+- AlanaHawke
+- Bryandagamer
+- GeForceFly - **Commentary Admin**
+- SeriouslyIntense
+- StarSketch
```
Comment (m4-used-rollout) [inline description/active/universal/15 Credits.md:165]: Shuffles2000 is missing

File: description/active/universal/15 Credits.md [hunk 5]
```diff
@@ -55,32 +54,33 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
...
 - MegamanOmega
+- Mmmmmmmmmmmmmmmmm
 - Moonbyes
+- Monstarules
 - MyRattata
-- Mmmmmmmmmmmmmmmmm
 - NonsensicalDreamer
 - Phaydra_
-- Pia Carrot
+- PIACarrot
```
Comment (m4-used-rollout) [inline description/active/universal/15 Credits.md:71]: Didn't you say PIA wanted to be temporarily removed? https://discord.com/channels/927292379436814406/928400707269320734/975532546383306762

**Review by m4-used-rollout — CHANGES_REQUESTED** [review]
Please give reasons for removing people

File: description/active/universal/15 Credits.md (hunk 1 above)
Comment (Monstarules) [reply description/active/universal/15 Credits.md:35]: Accident

File: description/active/universal/15 Credits.md (hunk 2 above)
Comment (Monstarules) [reply description/active/universal/15 Credits.md:96]: Replace-all accident

File: description/active/universal/15 Credits.md (hunk 3 above)
Comment (Monstarules) [reply description/active/universal/15 Credits.md:114]: Discussed in PM about Kipp, Lightning was accidentally removed as per prior comment.

File: description/active/universal/15 Credits.md (hunk 4 above)
Comment (Monstarules) [reply description/active/universal/15 Credits.md:165]: Fixed.

File: description/active/universal/15 Credits.md (hunk 5 above)
Comment (Monstarules) [reply description/active/universal/15 Credits.md:71]: Talked with him again, he's okay with remaining there now. It was a time-sensitive thing before we decided I would update the description.

**Pull request merged**